spring-cloud-dataflow-shell/pom.xml

```diff
@@ -13,6 +13,11 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-dataflow-shell-core</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
There are not tests in this module. I think we can remove this here.

Reply:
done